### PR TITLE
small changes in draws doc and quickstart vignette

### PR DESCRIPTION
**Pull request by nociale**
Addresses #341.

### CODE REVIEW

File: R/draws.R [hunk 1]
```diff
@@ -9,9 +9,9 @@
 #' in the absence of intercurrent events (ICEs) handled using reference-based imputation methods.
 #' For this reason, any observed outcome data after ICEs, for which reference-based imputation methods are
 #' specified, are removed and considered as missing for the purpose of estimating the imputation model, and for
-#' this purpose only. The imputation model is a mixed effects model repeated measures (MMRM) model that is valid
+#' this purpose only. The imputation model is a mixed model for repeated measures (MMRM) that is valid
```
Comment (gowerc) [inline R/draws.R:12]: Is "for" needed here? I thought it was just "Mixed model repeated measures" ?

File: vignettes/quickstart.Rmd [hunk 2]
```diff
@@ -49,7 +49,7 @@ data("antidepressant_data")
...
+We consider an imputation model with the mean change from baseline in the HAMD17 score as the outcome (variable `CHANGE` in the dataset). The following covariates are included in the imputation model: the treatment group (`THERAPY`), the (categorical) visit (`VISIT`), treatment-by-visit interactions, the baseline HAMD17 score (`BASVAL`), and baseline HAMD17 score-by-visit interactions. A common unstructured covariance matrix structure is assumed for both groups. The analysis model is an ANCOVA model with the treatment group as the primary factor and adjustment for the baseline HAMD17 score.
```
Comment (gowerc) [inline vignettes/quickstart.Rmd:52]: Not sure if I like this,  as in yes technically from an implementation point of view it is stored as a factor but thats not what we are trying to draw attention to here is it? As in isn't the point of this sentence trying to say that treatment group is the main variable of interest, I.e. the variable we are drawing inference on?

**Comment by gowerc** [comment]
@nociale ,

Just to say that the vignettes no longer automatically update and need to be manually re-run. I added a section to the contributing guide on how to do this. Please let me know if its not clear so I can adjust it :)

File: R/draws.R (hunk 1 above)
Comment (nociale) [reply R/draws.R:12]: I think that this is also what more commonly you find in the literature...

File: vignettes/quickstart.Rmd (hunk 2 above)
Comment (nociale) [reply vignettes/quickstart.Rmd:52]: I think that calling the treatment group the "primary factor" in an ANCOVA model is in line with the wording of the theory of standard ANOVA (e.g. https://en.wikipedia.org/wiki/Analysis_of_variance#For_a_single_factor). The previous wording wasn't unclear to the JOSS reviewers.

**Comment by gowerc** [comment]
LGTM

**Review by gowerc — APPROVED** [review]

**Pull request merged**